catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/src/main/webapp/templates/nodeRow.handlebars

```diff
@@ -22,7 +22,6 @@
     </a>
 
     <div class="modal-container">
-
```

Comment:
❓ Is this file changed at all? Looks like maybe just some white space. Let's remove it from the PR if that's the case.